### ISSUE TITLE
jQuery is not define

### ISSUE THREAD

**Issue by iShawnWang** [jquery/jquery-color#119]
I just `npm install jquery` then drag `jquery-color.min.js` into my project 
and import like this.

```
import $ from 'jquery'
import './jquery.color'
```

the error like this : 

```
app.bundle.js:19089 Uncaught ReferenceError: jQuery is not defined
    at Object.<anonymous> (app.bundle.js:19089)
    at __webpack_require__ (app.bundle.js:680)
    at fn (app.bundle.js:90)
    at Object.<anonymous> (app.bundle.js:5506)
    at __webpack_require__ (app.bundle.js:680)
```

also, why not update the npm package !!!

**Comment by mgol** [comment]
Which version of jQuery & jQuery Color do you use?

**Comment by iShawnWang** [comment]
jQuery latest 

```
jquery@^3.2.1:
  version "3.2.1"
  resolved "http://registry.npm.taobao.org/jquery/download/jquery-3.2.1.tgz#5c4d9de652af6cd0a770154a631bba12b015c787"
```

jQuery color from README : 

```
jQuery Color & Extended Names(previous two combined) [Compressed](http://code.jquery.com/color/jquery.color.plus-names-2.1.2.min.js)
```

**Comment by mgol** [comment]
Thanks for the report. This is fixed on `master` and will work once we release jQuery Color 3.0.0. Please follow #84 for progress.

**Comment by mgol** [comment]
jQuery Color `3.0.0-alpha.1` has been published to npm under the tag `beta`. It's also on the CDN: https://code.jquery.com/color/3.0.0-alpha.1/jquery.color.js.

Please test and report any issues before we release the final!

**Comment by mgol** [comment]
Released in [jQuery Color 2.2.0](https://github.com/jquery/jquery-color/releases/tag/2.2.0).

**Comment by G1King** [comment]
Cannot read properties of null (reading 'jquery')  
jquery-color@v2.2.0   jquery@1.10 
![Uploading image.png…]()
